_docs/setup/install-kubernetes.md

```diff
@@ -14,34 +14,37 @@ Quick Start instructions to install and configure Istio in a Kubernetes cluster.
 
 
 ## Prerequisites
+The following instructions assume you have access to a Kubernetes **1.7 or newer** cluster
+and  [RBAC (Role-Based Access Control)](https://kubernetes.io/docs/admin/authorization/rbac/) enabled.
+Always check the Release Notes of the Istio version you are installing for detailed requirements and limitations.
 
-* The following instructions assume you have access to a Kubernetes cluster with version > 1.7.0
-  and  [RBAC (Role-Based Access Control)](https://kubernetes.io/docs/admin/authorization/rbac/) enabled. To install Kubernetes locally,
+* Depending on your Kubernetes provider:
+  * For a local install, you can
```

Comment:
rephrase to "To install Istio locally, setup minikube" .. or some such variant, where the "you can" is eliminated.

Reply:
👍  done